src/reflect/scala/reflect/internal/util/Collections.scala

```diff
@@ -363,6 +363,50 @@ trait Collections {
     }
   }
 
+  private val TupleOfNil = (Nil, Nil)
+  final def partitionConserve[A](xs: List[A])(pred: A => Boolean): (List[A], List[A]) = {
```

Comment:
This looks like the same code added to the `List` class... could it be a firts draft, or is there a difference between them?

Reply:
Oops!

Reply:
`s/firts draft/farts draft/`